docs/understanding-airbyte/jobs.md

```diff
@@ -312,33 +312,33 @@ The Cloud Storage store is treated as the source-of-truth of execution state.
 
 The Container Orchestrator is only available for Airbyte Kubernetes today and automatically enabled when running the Airbyte Helm Charts deploys.
 
-```mermaid
----
-title: Start a new Sync
----
-sequenceDiagram
-%%    participant API
-    participant Temporal as Temporal Queues
-    participant Sync as Sync Workflow
-    participant ReplicationA as Replication Activity
-    participant ReplicationP as Replication Process
-    participant PersistA as Persistent Activity
-    participant AirbyteDB
-    Sync->>Temporal: Start a replication Activity
-    Temporal->>Sync: Pick up a new Sync
-    Temporal->>ReplicationA: Pick up a new task
-    ReplicationA->>ReplicationP: Starts a process
-    ReplicationP->>ReplicationA: Replication Summary with State message and stats
-    ReplicationA->>Temporal: Return Output (States and Summary)
-    Temporal->>Sync: Read results from Replication Activity
-    Sync->>Temporal: Start Persistent State Activity
-    Temporal->>PersistA: Pick up new task
-    PersistA->>AirbyteDB: Persist States
-    PersistA->>Temporal: Return output
-```
-
 Users running Airbyte Docker should be aware of the above pitfalls.
 
+## Worker V2
```

Comment:
@gosusnp here is where I would like you to focus on if you are short on time.

Once the blogpost is out, I'm going to link your post here at the bottom.

Reply:
LGTM, the only comment is on the `Worker V2` name, should we just refer to this as `Workloads` instead?